skafka/src/main/scala/com/evolutiongaming/skafka/consumer/Consumer.scala

```diff
@@ -34,6 +34,9 @@ trait Consumer[F[_], K, V] {
 
   def assignment: F[Set[TopicPartition]]
 
+  // TODO default `listener` to `RebalanceListener1.noOp` ?
+  //  or add subscribe methods without any listeners and hide implementation details
+  def subscribe(topics: Nes[Topic], listener: RebalanceListener1[F]): F[Unit]
```

Comment:
I propose to add two methods, in order to remove ambiguity

```
def subscribe(topics: Nes[Topic], listener: RebalanceListener1[F]): F[Unit]

def subscribe(topics: Nes[Topic]): F[Unit]
```

Reply:
agree

Reply:
fixed in bbd8c114c0e5a9efeb82e1f338b7055c26eb57b7